<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view</class>
 <widget class="QMainWindow" name="view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>997</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>997</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>997</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>3D viewer v2.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_25">
    <property name="geometry">
     <rect>
      <x>1651</x>
      <y>1041</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Screenshot</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1161</width>
      <height>861</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="download_obj">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>910</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open .obj</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_open">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>910</y>
      <width>871</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>590</y>
      <width>411</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QPushButton" name="change_move_y">
       <property name="text">
        <string>Move Y +</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="change_move_x">
       <property name="text">
        <string>Move X +</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="change_move_x2">
       <property name="text">
        <string>Move X -</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="dSpinBox_move">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="dSpinBox_move_2">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="dSpinBox_move_3">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="change_move_y2">
       <property name="text">
        <string>Move Y -</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="change_move_z2">
       <property name="text">
        <string>Move Z -</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="change_move_z">
       <property name="text">
        <string>Move Z +</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>880</y>
      <width>411</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="screenshot">
       <property name="text">
        <string>Screenshot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="start_image">
       <property name="text">
        <string>Start GIF</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stop_image">
       <property name="text">
        <string>Stop GIF</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>780</y>
      <width>411</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="background_white">
       <property name="text">
        <string>Background
 color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="change_vertex_color">
       <property name="text">
        <string>Vertex
 color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="change_edge_color">
       <property name="text">
        <string>Line
Color</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_of_inf_2">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>80</y>
      <width>367</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of polygons: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_of_inf">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>30</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of vertex: </string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>190</y>
      <width>411</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <widget class="QSpinBox" name="sBox_vertex_size">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Vertex Type</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="sBox_line_size">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cBox_vertex_type">
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Round</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Square</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Vertex Size</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Line Size</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Line Type</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cBox_line_type">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>NoPan</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SolidLine</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DashLine</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DotLine</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DashDotLine</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DashDotDotLine</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>420</y>
      <width>411</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="2">
      <widget class="QPushButton" name="change_rot_y2">
       <property name="text">
        <string>Rotation Y -</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="dSpinBox_rot">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="dSpinBox_rot_2">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="change_rot_x">
       <property name="text">
        <string>Rotation X +</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="change_rot_y">
       <property name="text">
        <string>Rotation Y +</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="change_rot_x2">
       <property name="text">
        <string>Rotation X -</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="change_rot_z">
       <property name="text">
        <string>Rotation Z +</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="change_rot_z2">
       <property name="text">
        <string>Rotation Z -</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="dSpinBox_rot_3">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>360</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="change_zoom">
       <property name="text">
        <string>Zoom +</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="dSpinBox_zoom">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="change_zoom_2">
       <property name="text">
        <string>Zoom -</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>130</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QRadioButton" name="radioButton">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Central</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Parellel</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
